app/scripts/ArrowheadDomainsTrack.js

```diff
@@ -249,7 +249,7 @@ class ArrowheadDomainsTrack extends TiledPixiTrack {
         : 0.4,
     );
 
-    graphics.alpha = this.options.rectangleDomainOpacity || 0.5;
+    // graphics.alpha = this.options.rectangleDomainOpacity || 0.5;
```

Comment:
Why is this commented out here? Does the track now support it?

Reply:
Oversight 😬  Commented out for testing and forgot to return.